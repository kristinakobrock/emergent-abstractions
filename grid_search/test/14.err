  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 26.08it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 27.67it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 24.42it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 22.13it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 24.25it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 25.71it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 25.67it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 25.66it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 25.59it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 24.50it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 27.37it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 29.32it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 27.94it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 26.07it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 25.88it/s]100%|############################################################################################| 50/50 [00:01<00:00, 24.51it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.60it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 19.75it/s] 38%|###################################7                                                         | 5/13 [00:00<00:00, 23.29it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 24.20it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 24.72it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.64it/s]
