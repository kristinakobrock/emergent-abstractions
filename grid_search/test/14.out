# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7408443093299866, "acc": 0.5680335760116577, "length": 2.750232458114624, "mode": "train", "epoch": 1}
{"loss": 0.6752661466598511, "acc": 0.60546875, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6646504998207092, "acc": 0.6226271390914917, "length": 1.0830036401748657, "mode": "train", "epoch": 2}
{"loss": 0.6590533256530762, "acc": 0.6236328482627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6596146821975708, "acc": 0.6235889792442322, "length": 1.0344326496124268, "mode": "train", "epoch": 3}
{"loss": 0.6589776873588562, "acc": 0.6207031607627869, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6585065722465515, "acc": 0.6239099502563477, "length": 1.0233315229415894, "mode": "train", "epoch": 4}
{"loss": 0.6580427289009094, "acc": 0.6238281726837158, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6588683128356934, "acc": 0.623818039894104, "length": 1.0169936418533325, "mode": "train", "epoch": 5}
{"loss": 0.6586800813674927, "acc": 0.623242199420929, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6590140461921692, "acc": 0.6235071420669556, "length": 1.0144253969192505, "mode": "train", "epoch": 6}
{"loss": 0.6581071615219116, "acc": 0.6230469346046448, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6585749387741089, "acc": 0.6227893233299255, "length": 1.0107758045196533, "mode": "train", "epoch": 7}
{"loss": 0.6589646339416504, "acc": 0.6244140267372131, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.658348023891449, "acc": 0.6231528520584106, "length": 1.0088533163070679, "mode": "train", "epoch": 8}
{"loss": 0.6582785248756409, "acc": 0.624218761920929, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6568134427070618, "acc": 0.624213695526123, "length": 1.0067020654678345, "mode": "train", "epoch": 9}
{"loss": 0.6593473553657532, "acc": 0.6248047351837158, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6582477688789368, "acc": 0.622346818447113, "length": 1.005113124847412, "mode": "train", "epoch": 10}
{"loss": 0.6595640182495117, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6581818461418152, "acc": 0.62347012758255, "length": 1.0037957429885864, "mode": "train", "epoch": 11}
{"loss": 0.6586398482322693, "acc": 0.6257812976837158, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6580957174301147, "acc": 0.6247521638870239, "length": 1.0034559965133667, "mode": "train", "epoch": 12}
{"loss": 0.6595524549484253, "acc": 0.6187500357627869, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6582728028297424, "acc": 0.6233591437339783, "length": 1.0031291246414185, "mode": "train", "epoch": 13}
{"loss": 0.6590235233306885, "acc": 0.6255859732627869, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6573678851127625, "acc": 0.6246424317359924, "length": 1.002130389213562, "mode": "train", "epoch": 14}
{"loss": 0.6593232750892639, "acc": 0.6246094107627869, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6576862335205078, "acc": 0.6248787045478821, "length": 1.0015968084335327, "mode": "train", "epoch": 15}
{"loss": 0.6587693691253662, "acc": 0.625, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6585403084754944, "acc": 0.6253461837768555, "length": 1.0012633800506592, "mode": "train", "epoch": 16}
{"loss": 0.660099983215332, "acc": 0.626953125, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.6582463383674622, "acc": 0.621482789516449, "length": 1.0015414953231812, "mode": "train", "epoch": 17}
{"loss": 0.6615594029426575, "acc": 0.612500011920929, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6583153009414673, "acc": 0.6228888630867004, "length": 1.0006918907165527, "mode": "train", "epoch": 18}
{"loss": 0.6600950956344604, "acc": 0.626171886920929, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6583547592163086, "acc": 0.6224207282066345, "length": 1.0006797313690186, "mode": "train", "epoch": 19}
{"loss": 0.6606894731521606, "acc": 0.6167969107627869, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.657747209072113, "acc": 0.6252334713935852, "length": 1.0005415678024292, "mode": "train", "epoch": 20}
{"loss": 0.6586927771568298, "acc": 0.6271483898162842, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6578017473220825, "acc": 0.6226567029953003, "length": 1.000519037246704, "mode": "train", "epoch": 21}
{"loss": 0.6595910787582397, "acc": 0.6277344226837158, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.6566841006278992, "acc": 0.6230092644691467, "length": 1.0004075765609741, "mode": "train", "epoch": 22}
{"loss": 0.6601698398590088, "acc": 0.6228516101837158, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6568971276283264, "acc": 0.6254047155380249, "length": 1.0009516477584839, "mode": "train", "epoch": 23}
{"loss": 0.6594825983047485, "acc": 0.6240234375, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6584147214889526, "acc": 0.6258777976036072, "length": 1.0002797842025757, "mode": "train", "epoch": 24}
{"loss": 0.6587123870849609, "acc": 0.6177734732627869, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.6571862101554871, "acc": 0.6245896220207214, "length": 1.0001975297927856, "mode": "train", "epoch": 25}
{"loss": 0.6615617871284485, "acc": 0.626171886920929, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.6564756631851196, "acc": 0.6246486306190491, "length": 1.0000828504562378, "mode": "train", "epoch": 26}
{"loss": 0.6589048504829407, "acc": 0.6244141459465027, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6558230519294739, "acc": 0.6255263686180115, "length": 1.0002162456512451, "mode": "train", "epoch": 27}
{"loss": 0.6598073244094849, "acc": 0.6181641221046448, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6569104790687561, "acc": 0.6240630149841309, "length": 1.000093936920166, "mode": "train", "epoch": 28}
{"loss": 0.6591534614562988, "acc": 0.6265625357627869, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6557793617248535, "acc": 0.6251758337020874, "length": 1.000135898590088, "mode": "train", "epoch": 29}
{"loss": 0.6594624519348145, "acc": 0.6222655773162842, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6558746695518494, "acc": 0.6250580549240112, "length": 1.0001457929611206, "mode": "train", "epoch": 30}
{"loss": 0.6590299606323242, "acc": 0.6279296875, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6558426022529602, "acc": 0.6251170635223389, "length": 1.0000388622283936, "mode": "train", "epoch": 31}
{"loss": 0.6601274609565735, "acc": 0.6255859732627869, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6570029854774475, "acc": 0.6258194446563721, "length": 1.000261664390564, "mode": "train", "epoch": 32}
{"loss": 0.6590539216995239, "acc": 0.6187500357627869, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.6558024287223816, "acc": 0.6234191656112671, "length": 1.0000076293945312, "mode": "train", "epoch": 33}
{"loss": 0.6598566174507141, "acc": 0.6275390982627869, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6561753153800964, "acc": 0.6249999403953552, "length": 1.0000072717666626, "mode": "train", "epoch": 34}
{"loss": 0.6603423953056335, "acc": 0.620898425579071, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.6553139686584473, "acc": 0.6248243451118469, "length": 1.0000044107437134, "mode": "train", "epoch": 35}
{"loss": 0.6591333746910095, "acc": 0.621874988079071, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6549891829490662, "acc": 0.625819981098175, "length": 1.0000406503677368, "mode": "train", "epoch": 36}
{"loss": 0.6597250699996948, "acc": 0.6273438334465027, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6559320092201233, "acc": 0.6259952783584595, "length": 1.0000211000442505, "mode": "train", "epoch": 37}
{"loss": 0.6604406237602234, "acc": 0.6218750476837158, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6557425260543823, "acc": 0.6259952783584595, "length": 1.0000057220458984, "mode": "train", "epoch": 38}
{"loss": 0.6598494052886963, "acc": 0.6228516101837158, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.6562400460243225, "acc": 0.625819742679596, "length": 1.0000274181365967, "mode": "train", "epoch": 39}
{"loss": 0.6607548594474792, "acc": 0.6279297471046448, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.655735969543457, "acc": 0.6250007152557373, "length": 1.0009307861328125, "mode": "train", "epoch": 40}
{"loss": 0.6625245809555054, "acc": 0.6156249642372131, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.656324565410614, "acc": 0.6241218447685242, "length": 1.0000079870224, "mode": "train", "epoch": 41}
{"loss": 0.6603309512138367, "acc": 0.6251952648162842, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6574472784996033, "acc": 0.6264052391052246, "length": 1.0000015497207642, "mode": "train", "epoch": 42}
{"loss": 0.6598328948020935, "acc": 0.6218750476837158, "length": 1.0, "mode": "test", "epoch": 42}
{"loss": 0.6567530035972595, "acc": 0.6261709332466125, "length": 1.0000042915344238, "mode": "train", "epoch": 43}
{"loss": 0.6623480916023254, "acc": 0.6167969107627869, "length": 1.0, "mode": "test", "epoch": 43}
{"loss": 0.6561621427536011, "acc": 0.6244731545448303, "length": 1.0000011920928955, "mode": "train", "epoch": 44}
{"loss": 0.6601808071136475, "acc": 0.6203124523162842, "length": 1.0, "mode": "test", "epoch": 44}
{"loss": 0.6551216244697571, "acc": 0.6258782148361206, "length": 1.0000001192092896, "mode": "train", "epoch": 45}
{"loss": 0.659384548664093, "acc": 0.626757800579071, "length": 1.0, "mode": "test", "epoch": 45}
{"loss": 0.6569142937660217, "acc": 0.6259366273880005, "length": 1.0000203847885132, "mode": "train", "epoch": 46}
{"loss": 0.661795973777771, "acc": 0.6253907084465027, "length": 1.0, "mode": "test", "epoch": 46}
{"loss": 0.6569410562515259, "acc": 0.6246485710144043, "length": 1.0, "mode": "train", "epoch": 47}
{"loss": 0.6609586477279663, "acc": 0.6167969107627869, "length": 1.0, "mode": "test", "epoch": 47}
{"loss": 0.6581072211265564, "acc": 0.6252341270446777, "length": 1.0000001192092896, "mode": "train", "epoch": 48}
{"loss": 0.6608759760856628, "acc": 0.6246094107627869, "length": 1.0, "mode": "test", "epoch": 48}
{"loss": 0.655495285987854, "acc": 0.6245315670967102, "length": 1.000000238418579, "mode": "train", "epoch": 49}
{"loss": 0.6606570482254028, "acc": 0.6257811188697815, "length": 1.0, "mode": "test", "epoch": 49}
{"loss": 0.656885027885437, "acc": 0.6257611513137817, "length": 1.0000020265579224, "mode": "train", "epoch": 50}
{"loss": 0.6608148813247681, "acc": 0.6191405653953552, "length": 1.0, "mode": "test", "epoch": 50}
{"loss": 0.6553981900215149, "acc": 0.6260538697242737, "length": 1.000000238418579, "mode": "train", "epoch": 51}
{"loss": 0.6612672805786133, "acc": 0.6275390982627869, "length": 1.0, "mode": "test", "epoch": 51}
{"loss": 0.6556470990180969, "acc": 0.6255270838737488, "length": 1.0000015497207642, "mode": "train", "epoch": 52}
{"loss": 0.6597956418991089, "acc": 0.6195312142372131, "length": 1.0, "mode": "test", "epoch": 52}
{"loss": 0.655699610710144, "acc": 0.6233607530593872, "length": 1.0000009536743164, "mode": "train", "epoch": 53}
{"loss": 0.6608848571777344, "acc": 0.624804675579071, "length": 1.0, "mode": "test", "epoch": 53}
{"loss": 0.6553463339805603, "acc": 0.6251755952835083, "length": 1.0000017881393433, "mode": "train", "epoch": 54}
{"loss": 0.6595813035964966, "acc": 0.6242188811302185, "length": 1.0, "mode": "test", "epoch": 54}
{"loss": 0.6568503379821777, "acc": 0.6269320845603943, "length": 1.0000007152557373, "mode": "train", "epoch": 55}
{"loss": 0.6653084754943848, "acc": 0.6087890267372131, "length": 1.0, "mode": "test", "epoch": 55}
{"loss": 0.6562127470970154, "acc": 0.6242387890815735, "length": 1.0000008344650269, "mode": "train", "epoch": 56}
{"loss": 0.65911865234375, "acc": 0.6279296875, "length": 1.0, "mode": "test", "epoch": 56}
{"loss": 0.6553651094436646, "acc": 0.6255267858505249, "length": 1.0000004768371582, "mode": "train", "epoch": 57}
{"loss": 0.6611198782920837, "acc": 0.6232421398162842, "length": 1.0, "mode": "test", "epoch": 57}
{"loss": 0.655925452709198, "acc": 0.6245896220207214, "length": 1.0000540018081665, "mode": "train", "epoch": 58}
{"loss": 0.6615390181541443, "acc": 0.6193360686302185, "length": 1.0, "mode": "test", "epoch": 58}
{"loss": 0.6569725871086121, "acc": 0.6269906759262085, "length": 1.000006914138794, "mode": "train", "epoch": 59}
{"loss": 0.6610777378082275, "acc": 0.6216797232627869, "length": 1.0, "mode": "test", "epoch": 59}
{"loss": 0.6568892598152161, "acc": 0.6255854368209839, "length": 1.0000004768371582, "mode": "train", "epoch": 60}
{"loss": 0.6609715819358826, "acc": 0.6214844584465027, "length": 1.0, "mode": "test", "epoch": 60}
test accuracy: 0.6019999980926514
